src/lib/layer.js

```diff
@@ -62,7 +62,10 @@ const defaultProps = {
   // Offset depth based on layer index to avoid z-fighting.
   // Negative values pull layer towards the camera
   // https://www.opengl.org/archives/resources/faq/technical/polygonoffset.htm
-  getPolygonOffset: ({layerIndex}) => [0, -layerIndex * 100]
+  getPolygonOffset: ({layerIndex}) => [0, -layerIndex * 100],
+
+  // Picking
+  pickingHighlightColor: NULL_PICKING_COLOR
```

Comment:
Again suggest simply using `null` as the default highlight color (unless we want highlighting on by default, in which case e.g. a transparent blue might be nice).